index.js

```diff
@@ -648,6 +659,23 @@ Client.prototype.busyP = function () {
   return busyP
 }
 
+Client.prototype.fetchVerifiedPublishers = function (callback) {
```

Comment:
Can we add an integrationTest or at least open a new issue to add one?

Reply:
absolutely, will add one